### ISSUE TITLE
Doc detail for custom rules miss

### ISSUE THREAD

**Issue by silverpark** [Respect/Validation#415]
Hi,

I would like build a custom Rules but it's not more specify in the documentation :
```
v::with('My\\Validation\\Rules\\');
v::myRule(); // Try to load "My\Validation\Rules\MyRule" if any
```

Can you explain how to build class for use ``myRules()`` ?

Regards

**Comment by henriquemoody** [comment]
Fixed by 845d976

Thanks for reporting! :panda_face:

**Comment by silverpark** [comment]
Thanks :+1:
